designdocs/batched_messages.md

```diff
@@ -188,15 +188,20 @@ of the `MessageResponse` to be more like the following:
 
 ```rust
 pub struct MessageResponse<AccountId, BlockNumber> {
-	#[cfg_attr(feature = "std", serde(with = "as_hex"))]
 	/// Serialized data in a the schemas.
-	pub payload: Payload,
-  /// Payload location
-  pub payload_location: PayloadLocation,
-	/// The public key of the provider and the signer of the transaction.
-	pub provider_key: AccountId,
-	/// Message source account id (the original source).
-	pub msa_id: MessageSourceId,
+	#[cfg_attr(feature = "std", serde(with = "as_hex_option", skip_serializing_if = "Option::is_none", default))]
```

Comment:
I think I might not be understanding this correctly. But it seems that when Option is None, we do not serialize.
Therefore, can we remove the Option from bytes params as shown here?https://github.com/LibertyDSNP/frequency/pull/477/files#diff-2d880531276d9fc6ad8edfe8cd913d301f52e900e3c5c3ee0c6fee5183276421R28?